make_profiler/lint_makefile.py

```
@@ -187,14 +226,9 @@ def main():
 
     targets, deps, deps_map = parse_targets(ast)
 
-    errors: List[str] = []
+    errors: List[LintError] = []
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)